lib/occupier/tenant.rb

```diff
@@ -57,28 +59,34 @@ def create!
     #
     def self.connect!(handle, environment = "development", logger = nil)
       client = Occupier::MongoMapper::Connection.new(environment, logger)
-      occupier = Occupier::Tenant.new(handle, client)
+      pg_client = Occupier::Postgres::Client.new(environment, nil)
+      occupier = Occupier::Tenant.new(handle, client, pg_client)
```

Comment:
Is `client` the mongo client? Is it worth it to rename it to be more explicit what kind of client we have?